templates/content-page/reports/a11y-1.json

```diff
@@ -0,0 +1,638 @@
+        "@type": ["earl:Assertor", "foaf:Person"],
+        "foaf:member": {
+          "@id": "_:wave_browser_extension",
+          "doap:name": "WAVE web accessibility evaluation tool, browser extension",
+          "doap:homepage": "https://wave.webaim.org/extension/",
+          "@type": [ "earl:Assertor", "earl:Software" ]
+        }
+      }
+    },
+
+    "dct:date": "2023-02-13",
+
+    "dct:description": "Partial evaluation focused only on the breadcrumb section part of the content page header. Evaluated all Success Criteria at Level A and AA of WCAG 2.1",
+    "acr:standard": "https://www.w3.org/TR/WCAG21",
+    "acr:conformanceOption": "act:standard/profiles/wcag#levelAA",
+
+    "earl:result": [
+      {
+        "@id": "_:sc_1.1.1",
+        "earl:test": "WCAG21:non-text-content",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:inapplicable",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "@id": "_:sc_1.2.1",
+        "earl:test": "WCAG21:audio-only-and-video-only-prerecorded",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:inapplicable",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "@id": "_:sc_1.2.2",
+        "earl:test": "WCAG21:captions-prerecorded",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:inapplicable",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "@id": "_:sc_1.2.3",
+        "earl:test": "WCAG21:audio-description-or-media-alternative-prerecorded",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:inapplicable",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "@id": "_:sc_1.2.4",
+        "earl:test": "WCAG21:captions-live",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:inapplicable",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "@id": "_:sc_1.2.5",
+        "earl:test": "WCAG21:audio-description-prerecorded",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:inapplicable",
+        "dct:description": "",
+        "earl:mode": "earl:semiAuto",
+        "@type": ["earl:TestResult", "earl:Assertion"]
+      },
+      {
+        "@id": "_:sc_1.3.1",
+        "earl:test": "WCAG21:info-and-relationships",
+        "earl:subject": "_:subject",
+        "earl:outcome": "earl:failed",
+        "dct:description": "Attached screen capture: Error P883 - SC 1.3.1, Nest heading properly. Heading elements must be ordered properly.",
```

Comment:
Add the text version of the image.

But wait a minute, is this description that is wrong or the attached image? Or is there 2 attachments?

The attached image say "Missing value for 'content' attribute. If either 'name', 'http-equiv', 'property', or 'itemprop' is specified, then the 'content' attribute must have a value